frontend/views/containers/proposals/ProposalVoteOptions.vue

```diff
@@ -107,11 +106,11 @@ export default ({
 
         if (oneVoteToPass(proposalHash)) {
           if (this.type === PROPOSAL_INVITE_MEMBER) {
-            passPayload = createInvite({
+            /* passPayload = createInvite({
               invitee: this.proposal.data.proposalData.member,
               creator: this.proposal.meta.username,
               expires: this.currentGroupState.settings.inviteExpiryProposal
-            })
+            }) */
```

Comment:
Whenever you're ready to clean up code, if you could address this please (just a reminder to not forget it)